okfn-brasil/serenata-website
# New Website - WIP

**Link to issue:**

- https://github.com/datasciencebr/serenata-website/issues/43

**How to test:**

1 - Clone the project and get into the folder:

```
$ git clone git@github.com:datasciencebr/serenata-website.git
$ cd serenata-website
```

2 - Check out the PR:

```
$ git checkout -b thulioph-next next
```

3 - Run the project:

```
$ bundle install
$ jekyll server
```

**Description of your solution:**

- Following the best practices of javascript code execution, we moved all the scripts tag to the footer with the [`async`](https://caniuse.com/#feat=script-async) attribute.

- I've split the SASS code into small parts, starting with `_variables.scss` that manages all CSS variables like, colors, font-definitions, etc. And `_grid.scss` that manages how the grid system works with Flexbox.
